janus_core/cli/md.py

```diff
@@ -27,6 +28,28 @@
 app = Typer()
 
 
+def _update_restart_files(summary: Path, restart_files: list[Path]) -> None:
+    """
+    Update restart files with final list.
+
+    Parameters
+    ----------
+    summary
+        Path to summary file.
+    restart_files
+        Restart files generated.
+    """
+    with open(summary, encoding="utf8") as file:
+        summary_info = yaml.safe_load(file)
+
+    summary_info["output_files"]["restarts"] = (
+        [str(file) for file in restart_files] if restart_files else None
```

Comment:
Would the empty list here be OK?

Reply:
Yep, we end up with `restarts: null` etc in the summary, which I think is fine